src/containers/FeedListViewerContainer.ts

```diff
@@ -0,0 +1,38 @@
+import { connect } from 'react-redux';
```

Comment:
This file is almost a copy of FeedListEditorContainer. I expect lot of development happening around these files in the near future and this may result in introducing subtle but important differences how they behave. Please consider different methods of reusing more code.